### ISSUE TITLE
Build failure using 1.1.0

### ISSUE THREAD

**Issue by jacogr** [jonchardy/typedoc-plugin-no-inherit#3]
Really excited for the 1.1.0 changes, however I'm getting this on CI, with 1.0.2 all is ok -

```
Loaded plugin /home/travis/build/polkadot-js/common/node_modules/typedoc-plugin-markdown
Loaded plugin /home/travis/build/polkadot-js/common/node_modules/typedoc-plugin-no-inherit
Using TypeScript 3.2.2 from /home/travis/build/polkadot-js/common/node_modules/typescript/lib
/home/travis/build/polkadot-js/common/node_modules/typedoc-plugin-no-inherit/dist/plugin.js:143
                var upLevel = extended.findReflectionByName(current.name);
                                       ^
TypeError: Cannot read property 'findReflectionByName' of undefined
    at NoInheritPlugin.isNoInheritUpHierarchy (/home/travis/build/polkadot-js/common/node_modules/typedoc-plugin-no-inherit/dist/plugin.js:143:40)
    at /home/travis/build/polkadot-js/common/node_modules/typedoc-plugin-no-inherit/dist/plugin.js:93:27
    at Array.forEach (<anonymous>)
    at NoInheritPlugin.onBeginResolve (/home/travis/build/polkadot-js/common/node_modules/typedoc-plugin-no-inherit/dist/plugin.js:90:39)
    at triggerEvents (/home/travis/build/polkadot-js/common/node_modules/typedoc/dist/lib/utils/events.js:129:43)
    at triggerApi (/home/travis/build/polkadot-js/common/node_modules/typedoc/dist/lib/utils/events.js:111:13)
    at eventsApi (/home/travis/build/polkadot-js/common/node_modules/typedoc/dist/lib/utils/events.js:21:18)
    at Converter.EventDispatcher.trigger (/home/travis/build/polkadot-js/common/node_modules/typedoc/dist/lib/utils/events.js:280:13)
    at Converter.resolve (/home/travis/build/polkadot-js/common/node_modules/typedoc/dist/lib/converter/converter.js:179:14)
    at Converter.convert (/home/travis/build/polkadot-js/common/node_modules/typedoc/dist/lib/converter/converter.js:114:28)
```

**Comment by jonchardy** [comment]
Ahh, somehow resolveType was returning undefined. Presumably there is something inheriting from outside the scope of your project.

This should be fixed with 1.1.1, just released.

One note: When testing with inheriting from HTMLElement, I noticed Typedoc isn't correctly identifying the static member HTMLElement.HTMLElement as inherited. Not sure what's going on there.

**Comment by jacogr** [comment]
Confirmed fixed in 1.1.1, thank you very much.
